pkg/log/logger_test.go

```diff
@@ -119,9 +121,9 @@ func TestInitializeLogger(t *testing.T) {
 
 	InitializeLogger(localLogger, &zapConfig)
 	assert.Equal(t, Logger(), localLogger)
-	// second initialization should not do anything
+	// second initialization should update
 	InitializeLogger(localLogger2, &zapConfig)
-	assert.Equal(t, Logger(), localLogger)
+	assert.Equal(t, Logger(), localLogger2)
```

Comment:
See comment earlier: this should not change anything as it can break the link between the k8shim and the core logger which breaks the ability to update the k8shim log level.

Reply:
As above, InitializeLogger() is how the shim will reconfigure the logger when the configmap changes. I've tested this locally in my shim, and it works as expected.

Reply:
Removed logging changes.